src/functions/getConfiguration/domain/config.mock.ts

```diff
@@ -5,13 +5,13 @@ import { Scope } from './scopes.constants';
 import { getGAId } from './getGAId';
 
 const generateAllowedTestCategories = (env: string): string[] => {
-  return env === Scope.DEV ? ['B'] : [];
+  return [Scope.PERF, Scope.PROD, Scope.UAT].includes(env as Scope) ? [] : ['B'];
```

Comment:
PROD or LIVE?

Reply:
Was just copying the feature flag below (`enableLogoutButton`). The constants file reads as follows:

```
export enum Scope {
  DEV = 'dev',
  PROD = 'live',
  STAGING = 'staging',
  UAT = 'uat',
  PERF = 'perf',
}
```

Reply:
include UAT?